pyproject.toml

```diff
@@ -0,0 +1,24 @@
+[tool.black]
+line-length = 88
+target-version = ["py311"]
+
+[tool.isort]
+profile = "black"
+line_length = 88
+known_first_party = ["odh_data_processing"]
+src_paths = ["kubeflow-pipelines"]
+
+[tool.ruff]
+line-length = 88
+target-version = "py311"
+extend-exclude = [
+  "docs/_build",
+  "notebooks/vendor",
+  "data",
+  "artifacts",
+  ".ipynb_checkpoints",
+  ".venv",
+  "venv",
+]
+lint.select = ["E","F","I","B","W","UP"]
```

Comment:
Can you add comments to the `lint.select` and `lint.ignore` sections just like they have here: https://github.com/Red-Hat-AI-Innovation-Team/sdg_hub/blob/main/pyproject.toml#L114 so that it's easy for someone in the future to know which options are ignored and what this line does?